### ISSUE TITLE
[Bug]: Cannot run checks using -g <group_name>

### ISSUE THREAD

**Issue by nojanath** [toniblyx/prowler#1290]
**What happened?**  
Running checks with -g <group_name> does not work. If I omit the -g option, prowler works great. I can also specify individual checks, e.g. -c check11, and prowler works as expected.

Running with -g group1, for example, the prowler banner is shown along with the credential report. The last line is the Caller Identity ARN and then prowler exits with no error. I tested the following groups and all failed.

- group1
- group2
- group3
- group4
- cislevel1
- cislevel2
- extras

**How to reproduce it**  
```
git clone https://github.com/prowler-cloud/prowler.git
cd prowler
./prowler -p profile-name -g group1
```

Running prowler against a single account.

**Expected behavior**  
I expected a specific set of checks to be run according to the chosen group.

**From where are you running Prowler?**  
Please, complete the following information:
 - Resource: MacBook
 - OS: macOS 11.6.7
 - AWS-CLI Version [`aws --version`]: 2.6.3
 - Prowler Version [`./prowler -V`]: 2.11.0-21July2022
 - Shell and version: zsh 5.8, I tried bash 3.2.57 as well

**Additional context**
Workaround is to use Prowler 2.10.0. The group option works with that version.

**Comment by MrCloudSec** [comment]
Hi @nojanath , thank you for reaching us out!
Please, could you please copy here the logs by executing it with `bash -x` or just send us the unexpected behavior? 
I see no errors in my environment when running group1.

**Comment by nojanath** [comment]
This seems to be something weird on my work MacBook. I created a new user profile and still had the same problem. However, I tried my personal MacBook and it worked as expected.

The output of `bash -x` is attached.

[prowler-output.txt](https://github.com/prowler-cloud/prowler/files/9223720/prowler-output.txt)

**Comment by jfagoagas** [comment]
Hi @nojanath, we can't reproduce your issue and I suspect it is related to your bash version.

Remember that you need `bash` to run Prowler, `zsh` is not supported. Test it again with the latest version available of bash in your workstation ant let us know if everything works fine.

Thanks!

**Comment by williambrady** [comment]
Apple may be the culprit as they bundle an old version of `bash`. `bash 3.2.57` is years old and may be missing support for some of the options being used in current versions of prowler. `bash` is current running in the 5.x line, so upgrading may resolve this issue. 

While Apple doesn't support newer versions of `bash`, there are tons of online resources to assist with upgrading.

**Comment by nojanath** [comment]
I tried bash 5.1.16, but the problem still exists. Bash 3.2.57 is indeed very old but it does actually work on my personal laptop. No problems with prowler at all.

I'm going to chalk this up to my company-controlled laptop and whatever weird 3rd party software they have installed. I can workaround it by running Prowler 2.10.0.

 Thanks for all the work on Prowler. Looking forward to version 3! Closing this out.
